examples/print_match_text.rs

```diff
@@ -1,12 +1,19 @@
 use clap::Parser;
-use tree_sitter_grep::{run_with_callback, Args};
+use tree_sitter_lint_tree_sitter_grep::{run_with_callback, Args};
 
 fn main() {
     let args = Args::parse_from(["tree_sitter_grep", "-q", "(function_item) @f"]);
-    run_with_callback(args, |node, file_contents, path| {
+    run_with_callback(args, |query_match, file_contents, path| {
```

Comment:
As of now I updated all of the crate-level "callback"-oriented entry points to use the match-oriented (vs single-capture-oriented) querying/callback style

Not sure where it should actually "land", for the moment mostly just driving based on the `tree-sitter-lint` use case